verification_token/models.py

```diff
@@ -93,8 +114,11 @@ def generate_key(cls, generator=None, *args, **kwargs):
     @property
     def is_valid(self):
         return (
-            self.is_active and self.key and self.expiration_in_minutes and
-            timezone.now() <= self.created_at + timedelta(minutes=self.expiration_in_minutes)
+            self.is_active and self.key and (
+                (self.expiration_in_minutes and timezone.now() <= self.created_at
+                 + timedelta(minutes=self.expiration_in_minutes)
+                 ) or self.expiration_in_minutes is None
+            )
```

Comment:
self.is_active and sel.key and (self.expiration_in_minutes is None or timezone.now() <= self.created_at + timedelta(minutes=self.expiration_in_minutes))

Reply:
Pravda, ta část podmínky byla zbytečná.